### ISSUE TITLE
Strategy for creating validation data from FER2013 dataset(or other dataset)

### ISSUE THREAD

**Issue by as43647** [ArnabKumarRoy02/ResEmoteNet#28]
Hi, I noticed that the FER2013 dataset doesn't include a dedicated validation set. Currently, I'm randomly selecting 20% of the training data to create a validation set.

However, my current experiment result on this setup is 63.4, which I find quite low. That's why I'm wondering if this approach to creating a validation set is suboptimal.

Is this a recommended strategy? Or is there a better way to create a validation set from this dataset? Thank you.

**Comment by ArnabKumarRoy02** [comment]
Hey @as43647, you can find a copy of the original FER2013 dataset [here](https://www.kaggle.com/datasets/deadskull7/fer2013). This contains **Training** (Training set), **PublicTest** (Validation set), and **PrivateTest** (Test set). I believe this would help you.

To your question: "Is this a recommended strategy? Or is there a better way to create a validation set from this dataset?", I would say, try to find out if any validation set exists for that particular dataset. If such doesn't exist then, try to make one using some portion of the training set.
